dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -1828,9 +1828,9 @@ RtpsUdpDataLink::RtpsReader::gather_ack_nacks_i(MetaSubmessageVec& meta_submessa
       wi->second.ack_pending_ = false;
 
       SequenceNumber ack;
-      CORBA::ULong num_bits = 1;
+      CORBA::ULong num_bits = 0;
       LongSeq8 bitmap;
-      bitmap.length(1);
+      bitmap.length(0);
```

Comment:
This line and the line below should be removed.  The default-constructed sequence already has length 0, and indexing position 0 is undefined behavior.

Reply:
Update completed